model/patch_lifecycle_test.go

```diff
@@ -1158,3 +1158,56 @@ func TestAbortPatchesWithGithubPatchData(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigurePatch(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(VersionCollection, patch.Collection))
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	id := mgobson.NewObjectId()
+	v := &Version{
+		Id:        id.Hex(),
+		Status:    evergreen.VersionStarted,
+		Activated: utility.TruePtr(),
+	}
+	assert.NoError(t, v.Insert())
+	p := &patch.Patch{
+		Id:         id,
+		Version:    v.Id,
+		Status:     evergreen.VersionStarted,
+		Activated:  true,
+		Project:    "project",
+		CreateTime: time.Now().Add(-time.Hour),
+		GithubPatchData: thirdparty.GithubPatch{
+			BaseOwner: "owner",
+			BaseRepo:  "repo",
+			PRNumber:  12345,
+		},
+		Tasks: []string{"my_task"},
+		VariantsTasks: []patch.VariantTasks{
+			{
+				Variant: "my_variant",
+				Tasks:   []string{"my_task"},
+			},
+		},
+	}
+	assert.NoError(t, p.Insert())
+	pRef := &ProjectRef{
+		Id: mgobson.NewObjectId().Hex(),
+	}
+	req := PatchUpdate{
+		Description: "updating the description only!",
+	}
+	pp := ParserProject{
+		Id: id.Hex(),
+	}
+	assert.NoError(t, pp.Insert())
+	_, err := ConfigurePatch(ctx, evergreen.GetEnvironment().Settings(), p, v, pRef, req)
```

Comment:
Use `mock.Environment` instead of the global environment if possible when testing with the env (unless you need to depend on the global testing env for some reason), or just pass in an empty `evergreen.Settings{}` if the test doesn't depend on any particular settings (seems like it doesn't).